prusti-specs/src/specifications/common.rs

```diff
@@ -127,38 +127,3 @@ fn generate_name_for_type(ty: &syn::Type) -> Option<String> {
         }
     }
 }
-
-
-#[cfg(test)]
-mod tests {
```

Comment:
@vl0w FYI I deleted this: I didn't want regex as a dependency (even as `dev-dependency` it somehow shows up in `Cargo.lock` of the cargo test…), and this test isn't particularly useful. If we really want to test stability of the mangled names, it can be a UI test.

Reply:
That's ok. Afaik the other tests (also in the other PR) don't use Regex.